docs/Catalog/Preview.md

```diff
@@ -29,14 +29,27 @@ The following file formats are supported:
 * .ent
 * .pdb
 
+## Image previews
+
+The Quilt Catalog uses a [Lambda
+function](https://github.com/quiltdata/quilt/tree/master/lambdas/thumbnail)
+to automatically generate thumbnail previews of common image file
+formats and specific microscopy image file formats.
+
+### Known limitations
+
+Automated previews of 8-bit depth and higher image files are not
+currently supported.
+
 ## Binary and special file format previews
 
-* Excel (.xls, .xlsx)
 * FCS Flow Cytometry files (.fcs)
-* Images (.gif, .jpg, .png, .tif, .tiff, etc.)
```

Comment:
Sorry if I confused you hear but why are we eliminating the images line? We should not.
You should note .tiff (incl. OME TIFF) and .czi — which are the two microscopy formats we support per your opening sentence above.

Reply:
Because we now have a dedicated `Image previews` section and the AICS library supports many, many file formats - not just what we have listed. I can specifically add OME TIFF and .czi, but it might be more simple to link to the pinned versions of AICS GH-based docs with all the formats supported?

Reply:
For now, I've reinstated the images line because AICSImageIO relies on ImageIO for common image formats and that list changes often.